elide-core/src/test/java/com/yahoo/elide/core/PersistentResourceTest.java

```diff
@@ -110,6 +117,7 @@ public PersistentResourceTest() {
         badUserScope = new RequestScope(null, null, mock(DataStoreTransaction.class),
                 new User(-1), null, elideSettings);
         reset(goodUserScope.getTransaction());
+        assertFalse(goodUserScope.isUseFilterExpressions());
```

Comment:
This is not a test.  Why use assert here?

Reply:
Why is this not a test?